src/Integration/Telemetry/TelemetryManager.cs

```diff
@@ -140,7 +140,7 @@ private void EnableAllEvents()
         private TelemetryPayload GetPayload(TelemetryData telemetryData)
         {
             return TelemetryHelper.CreatePayload(telemetryData, DateTimeOffset.Now,
-                solutionBindingTracker.CurrentConfiguration.Mode != NewConnectedMode.SonarLintMode.Standalone);
```

Comment:
I changed the _CreatePayload_ method to take the binding as a parameter, otherwise the caller would have to work out _isConnected/isSonarCloud_.